src/Microsoft.DotNet.Interactive.Http/ClearValues.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.Http;
+
+public class ClearValues : KernelCommand
```

Comment:
This seems generally useful. Any reason why this is not defined in the main assembly?

Currently the http editor does not (need to) directly reference the `Interactive.Http` assembly. Adding that reference may not be problematic - but still wanted to double check whether we can move this to the main assembly and avoid the additional dependency.